src/blueprints/tv2/value-objects/tv2-layers.ts

```diff
@@ -33,3 +40,27 @@ export enum Tv2SisyfosLayer {
   AUDIO_BED = 'sisyfos_source_audiobed',
   RESYNCHRONIZE = 'sisyfos_resync'
 }
+
+export enum Tv2SwitcherMixEffectLayer {
+  PROGRAM = 'me_program',
+  CLEAN = 'me_clean',
+  CLEAN_USK_FULL = 'clean_usk_full',
+  CLEAN_USK_EFFECT = 'clean_usk_effect',
+  NEXT = 'me_next',
+  NEXT_JINGLE = 'me_next_jingle'
+}
+
+export enum Tv2SwitcherAuxLayer {
+  PROGRAM = 'aux_pgm',
+  CLEAN = 'aux_clean',
+  MIX_EFFECT_3 = 'aux_mix_effect_3', // AUX set by Sofie, but the M/E is uncontrolled by Sofie
+  WALL = 'aux_wall',
+  AR = 'aux_ar',
+  VIZ_OVL_IN_1 = 'aux_viz_ovl_in_1',
+  VENUE = 'aux_venue',
+  LOOKAHEAD = 'aux_lookahead',
+  DVE = 'aux_dve',
+  VIDEO_MIX_MINUS = 'aux_video_mix_minus',
+  SCREEN = 'aux_screen',
+  SERVER_LOOKAHEAD = 'aux_server_lookahead'
+}
```

Comment:
These already have Enums. 
You are currently only using the `Tv2SwitcherMixEffectLayer.PROGRAM`, `Tv2SwitcherMixEffectLayer.CLEAN` and `Tv2SwitcherAuxLayer.LOOKAHEAD`.
The only one of these that is missing is the `CLEAN`. The rest are in `Tv2AtemLayer.PROGRAM` and `Tv2AtemLayer.LOOKAHEAD`.

So you should delete these again and add `Tv2AtemLayer.CLEAN`.

Reply:
Fixed.